src/Reflectivity-Tools/IconStylerRealTime.class.st

```diff
@@ -0,0 +1,239 @@
+"
+I am a copy of the class IconStyler because I use my sub-classes to stylize the method tab and I just want to use a few select ones.
+"
+Class {
+	#name : #IconStylerRealTime,
+	#superclass : #RBProgramNodeVisitor,
+	#instVars : [
+		'textModel'
+	],
+	#category : #'Reflectivity-Tools-RealTimeStyling'
+}
+
+{ #category : #options }
+IconStylerRealTime class >> shouldStyle [
+	^true
+]
+
+{ #category : #style }
+IconStylerRealTime class >> style: textModel [
+	^self new textModel: textModel.
+]
+
+{ #category : #api }
+IconStylerRealTime class >> styleText: textModel withAst: ast [
```

Comment:
I believe we can avoid separate styler hierarchy. We can have a method on IconStyler like:
```Smalltalk
IconStyler>>styleTextForRealTimeHighlighting: textModel withAst: ast
	self allSubclassesDo: [ :styleClass | 
			styleClass shouldStyleTextInRealTime ifTrue: [...]]
```
Then we just need to decide which classes should return true from #shouldStyleTextInRealTime.